doc/tutorial/describing-code.rst

```diff
@@ -229,3 +232,43 @@ And finally, ``make test`` reports success!
 For big projects though, this manual approach can become a bit tedious.
 In the next section, you will see :doc:`how to automate the
 process </tutorial/automatic-doc-generation>`.
+
+Other languages (C, C++, others)
+--------------------------------
+
+Documenting and cross-referencing objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sphinx also supports documenting and cross-referencing objects written in
+other programming languages. There are four extra built-in domains:
+C, C++, JavaScript, and reStructuredText, and third party extensions may
+define domains for more languages, such as
+`.NET <https://sphinxcontrib-dotnetdomain.readthedocs.io>`,
```

Comment:
I'm not sure if is worth mentioning the .NET extension since I think it's deprecated/not maintained.

Reply:
I think such deprecation hasn't been announced yet? But sure, I can remove it.